### PR TITLE
Prac4

### PR DESCRIPTION
**Pull request by smiling-demon**

### CODE REVIEW

**Comment by Smart781** [comment]
Добрый вечер! 
В compute_td_target нужно проводить вычисления под with torch.no_grad(), также при вычислении target нужно учитывать переменную term, потому что итог должен быть равен нулю в случае если эпизод завершился.
Также в sample_prioritized_batch нужно получать вероятности с помощью softmax и  symlog.
В run_dqn_prioritized_rb нужно вычислать под with torch.no_grad().
Исправления можете прислать до воскресенья 23:59.

**Comment by Smart781** [comment]
Добрый день! Зачтено.
